task/task_smoke_test.go

```diff
@@ -40,9 +89,17 @@ func TestTaskSmoke(t *testing.T) {
 	}
 
 	if testName == "" {
-		testName = "smoke test"
+		testName = fmt.Sprintf("smoke test: %s", targetName)
 	}
-
+	script := `
+          #!/bin/sh -e
+          [ -n "$TEST_GPU" ] && nvidia-smi
```

Comment:
Even [subjectively] better?
```suggestion
          test -v TEST_GPU && nvidia-smi
```
or
```suggestion
          [ -v TEST_GPU ] && nvidia-smi
```